ada/language/ast.py

```diff
@@ -3227,6 +3227,32 @@ def next_part():
 
         )
 
+    @langkit_property(return_type=Bool,
+                      dynamic_vars=[default_origin()], public=True)
+    def is_definite_subtype():
+        """
+        Returns whether this is a definite subtype.
+        """
+        return Entity.match(
+            lambda itd=T.IncompleteTypeDecl: False,
```

Comment:
Well I have to say I'm not sure: You can't declare a variable of one, so does it make sense to say that it is definite or not ?

Reply:
Haha, good question! Is the notion of “definite subtype” specified in the RM? If not, it would be worth 1) asking Fedor what he really needs for this case 2) define what “definite subtype” means in the docstring.

Reply:
I'm pretty sure that Fedor doesn't care about that use case, but I'm also pretty sure it's defined in the ARM :) Will ask around. We want this to fit the ARM definition pretty closely.